docs/source/conf.py

```diff
@@ -170,3 +171,6 @@
 }
 
 autoclass_content = "both"
+
+#  -- Options for sphinxcontrib.bibtex ------------------------------------
+bibtex_bibfiles = ['demos/demo_references.bib']
```

Comment:
Nice. So far we haven't used proper links for the references but probably should.